app/mailers/devise_mailer.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class DeviseMailer < GovukNotifyRails::Mailer
+  include Devise::Controllers::UrlHelpers
+  include ActionView::Helpers::DateHelper
+
+  def reset_password_instructions(user, token, _opts = {})
+    set_template('7314475f-62b0-41bf-8aff-768427cba456')
```

Comment:
extract template ids to method, constant or mixin, can pair on this:

```
  # example only
  GOVUK_NOTIFY_TEMPLATES = {
    reset_password_instructions: 'that-long-id',
    emailed_changed: 'that-long-id',
   }
```
and use
```
set_template(GOVUK_NOTIFY_TEMPLATES[:reset_password_instructions))

```

Reply:
👍